bigbluebutton-html5/imports/ui/components/end-meeting-confirmation/service.js

```diff
@@ -0,0 +1,36 @@
+import Users from '/imports/api/users';
+import Auth from '/imports/ui/services/auth';
+
+const USERS_TO_DISPLAY = Meteor.settings.public.app.maxUsersForConfirmation;
+
+const userFindSorting = {
+  emojiTime: 1,
+  role: 1,
+  phoneUser: 1,
+  sortName: 1,
+  userId: 1,
+};
+
+const getUsers = () => {
```

Comment:
You're duplicating lots of code from user-list/service.js Did you try to just import the methods from there?

Reply:
Yes, initially I did try to simply use it but I felt uncomfortable doing so for two reasons: 1. I'm not sure if it's OK to import a service from an unrelated component 2. I didn't need a large portion of the code that seems to deal with locked users. I'd like to, of course, duplicate as little code as possible, though so perhaps you may have other suggestions.